src/db/oidc.rs

```diff
@@ -0,0 +1,36 @@
+use serde::{Deserialize, Serialize};
+
+use openidconnect::core::{
+    CoreAuthDisplay, CoreClaimName, CoreClaimType, CoreClientAuthMethod, CoreGrantType,
+    CoreJsonWebKey, CoreJsonWebKeyType, CoreJsonWebKeyUse, CoreJweContentEncryptionAlgorithm,
+    CoreJweKeyManagementAlgorithm, CoreJwsSigningAlgorithm, CoreResponseMode, CoreResponseType,
+    CoreSubjectIdentifierType,
+};
+use openidconnect::{AdditionalProviderMetadata, CsrfToken, Nonce, ProviderMetadata};
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RevocationEndpointProviderMetadata {
+    pub revocation_endpoint: String,
+}
+impl AdditionalProviderMetadata for RevocationEndpointProviderMetadata {}
```

Comment:
I think we can move the contents of this file into the root of the project (`/oidc.rs`) since the structs and types in this file are not stored in the database.